<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomWindow</class>
 <widget class="QWidget" name="RoomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="rommName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Chat Room</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="roomLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="userList">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>121</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="chatresource.qrc"/>
 </resources>
 <connections/>
</ui>
